dtests/src/test/java/io/snappydata/hydra/rowTableCPP/rowTableCPPTest.conf

```diff
@@ -0,0 +1,26 @@
+// INCLUDE $JTESTS/io/snappydata/hydra/connectionPool/connPool.inc;
+
+INCLUDE $JTESTS/io/snappydata/hydra/northwind/startEmbeddedModeCluster.conf;
+
+INCLUDE $JTESTS/io/snappydata/hydra/rowTableCPP/serverHA.inc;
+INCLUDE $JTESTS/io/snappydata/hydra/northwind/backUpServerConfig.inc;
+
+INITTASK   taskClass   = io.snappydata.hydra.cluster.SnappyTest taskMethod  = HydraTask_executeSQLScripts
+           io.snappydata.hydra.cluster.SnappyPrms-sqlScriptNames = create_and_load_partitionedRowTables_persistent.sql
+           io.snappydata.hydra.cluster.SnappyPrms-dataLocation = ${dataFilesLocation}
+           threadGroups = snappyThreads
+           ;
+
+TASK       taskClass   = io.snappydata.hydra.cluster.SnappyTest taskMethod  = HydraTask_executeScripts
+           io.snappydata.hydra.cluster.SnappyPrms-scriptNames = ${scriptLocation}
+           io.snappydata.hydra.cluster.SnappyPrms-scriptArgs = ${timeduration},${scriptPWD}
+           threadGroups = snappyStoreThreads
+           maxThreads = 1
+           ;
+
+hydra.Prms-totalTaskTimeSec           = 900;
+hydra.Prms-maxResultWaitSec           = 1200;
+io.snappydata.hydra.cluster.SnappyPrms-isCppTest = true;
+/*
+
```

Comment:
Please add CLOSETASK/ENDTASK to stop the cluster. You need to add the existing conf for it:

INCLUDE $JTESTS/io/snappydata/hydra/northwind/stopEmbeddedModeCluster.conf;

Reply:
Added